### ISSUE TITLE
Pulley drawer content can lock all events programmatically?

### ISSUE THREAD

**Issue by willianB** [52inc/Pulley#323]
Hello.
I am trying to block / unblock all events programmatically. How can I do it?

**Comment by brendan09** [comment]
What kind of events?

**Comment by willianB** [comment]
What kind of events? 

For example, events that allow expansion to collapse open. In a few words, the drawer content can not be expanded.

**Comment by brendan09** [comment]
Override the drawer delegate and return only 1 supported drawer position in the appropriate delegate. When changing what is returned there make sure and call the appropriate method to notify pulley that you updated the supported drawer positions.

**Comment by willianB** [comment]
Will this work at runtime?

**Comment by brendan09** [comment]
Yes. It can be changed whenever you want as long as you call the method on Pulley to tell it that it changed.

**Comment by willianB** [comment]
OK thank you for your time. I'll try

**Comment by willianB** [comment]
Perfect, it worked correctly. Thank you very much, greetings from Colombia.

**Comment by Jemarlwecorp** [comment]
how did you do the above. Kind of lost, trying to achieve the exact same thing.
